pipelines/metrics_pipeline_tract_multi_test.yaml

```diff
@@ -0,0 +1,5 @@
+description: Full metrics pipeline
+inherits:
+#  - location: $METRIC_PIPELINE_TASKS_DIR/pipelines/gen_inputs_matched_multi_tract.yaml
```

Comment:
Maybe we want to remove the commented out lines of code here, since they seem to be leftovers from an earlier concept to using multi-band matched catalogs